server.js

```diff
@@ -1,38 +1,70 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+
+import booksData from "./data/books.json";
+
+//Author model and connection to database
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-books";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const port = process.env.PORT || 8080;
+const app = express();
+
+//A model
+const Author = mongoose.model("Author", {
```

Comment:
Since this is a database containing different book objects, I think it would make more sense to name the main model here Book instead of Author 📚
It can be confusing, since with this name I am expecting to get data on a bunch of different authors, but for this data set this is not the case.

Then in that case, if you have a main Book model instead, you could take advantage of all the information that is provided to us in the dataset, and add more properties to this model, like: "average_rating", "num_pages", "ratings_count", etc. 🤓

Reply:
Cool thank you for the tips!

Reply:
@VanessaSue27 is right. Authors collection should contain the list of authors, not books. When you are designing your data you need to ask yourself a question "what I will store here".

You can create separate Authors collection for example like this:
{ id: String, name: String, books: []}